pkg/ingester/shipper.go

```diff
@@ -197,7 +198,7 @@ func (s *Shipper) Sync(ctx context.Context) (shipped int, err error) {
 // This updated version of meta.json is however not persisted locally on the disk, to avoid race condition when TSDB
 // library could actually unload the block if it found meta.json file missing.
 func (s *Shipper) upload(ctx context.Context, meta *metadata.Meta) error {
-	level.Info(s.logger).Log("msg", "upload new block", "id", meta.ULID)
+	level.Info(s.logger).Log("msg", "uploading new block", "id", meta.ULID)
```

Comment:
Maybe we can move this to `Sync` method just before `s.upload(ctx, m)` call, to have both logs close to each other.